solutions/src/test/kotlin/de/thermondo/solutions/basics/Basics07SolutionTest.kt

```diff
@@ -0,0 +1,19 @@
+package de.thermondo.solutions.basics
+
+import de.thermondo.test.utils.BasePrintlnTest
+import org.junit.jupiter.api.Test
+import kotlin.test.assertNotNull
+
+class Basics07SolutionTest : BasePrintlnTest() {
```

Comment:
```suggestion
class Basics07SolutionTest {
```

No need to extend here.